### ISSUE TITLE
CKPX does not recognize all files in gdrive

### ISSUE THREAD

**Issue by gator73** [subdavis/CKPX#18]
Just installed CKPX and running through the setup. App did not see any of my database files on gdrive. I have been using the Keepass Google Drive Sync plugin for several years and have multiple databases on gdrive. While on the gdrive site I "made a copy" of my most used db file and bingo, CKPX immediately found the "copy of" file. While it found the "copy of " file, this would create more work for me as that negates the functionality of the sync plugin.
Upon closer review, it appears none of the files created by KP google sync plugin show up in CKPX. What am I missing regarding the "synced" db file?
I appreciate any ideas or help.

**Comment by gator73** [comment]
OK, so this occurred to me just after posting. In reviewing the details on gdrive, I noticed all the gsync files showed "created by google sync plugin" whereas the "copy of" showed I created it or created on gdrive. So I deleted the file created by gsync plugin and immediately uploaded the file to gdrive using file explorer. Now the file shows I created it. Tried the gsync keepass plugin and the sync worked. Triedk CKPX and it FOUND the file and allowed me to use it. Who knew?
